src/components/Payment/PayPal/PayPal.tsx

```diff
@@ -79,21 +83,24 @@ const PayPal = ({
           )}
       </CopyStyled>
       <CheckboxWrapperStyled>
-        <CheckboxLegacy
-          className='adyen-checkout__bank-checkbox checkbox-paypal'
-          checked={isChecked}
-          onClickFn={(e) => {
-            e.target.parentElement.classList.remove(
+        <Checkbox
+          className='adyen-checkout__bank-checkbox checkbox-wrapper-paypal'
+          id='checkbox-paypal'
```

Comment:
i think the optional `?` is redundant here, since you passed the `| undefined` type